community/kernel/src/test/java/org/neo4j/kernel/impl/transaction/log/pruning/TestLogPruning.java

```diff
@@ -97,7 +98,7 @@ public void pruneByFileSize() throws Exception
         // Given
         int size = 1050;
         int logThreshold = transactionLogSize * 3;
```

Comment:
This test's name is `pruneByFileSize`, but after we changed the `doTransaction` method, I am a bit surprised to see the assertion `assertTrue( logFileSize >= size - logThreshold && logFileSize <= size + logThreshold );` passed. Also the assertion itself is strange too.